### DISCUSSION TITLE
Why isn't my GitHub Actions workflow triggering after a PR merge?

### DISCUSSION THREAD

**Discussion by vighneshb04** [community/community#136947] [Discussions] [answered]
### Select Topic Area

Question

### Body

 My GitHub Actions workflow is supposed to trigger when a PR is merged into the main branch. It was working fine until recently, but now the workflow doesn't run at all after a PR merge. I've checked the workflow YAML file, and it seems correct. What could be causing this issue?

**Comment by vighneshb04** [comment, marked as answer]
Check Workflow Trigger Configuration:

Ensure that your workflow YAML file has the correct event triggers set up. For PR merges, you should have something like this in your workflow file:
yaml
Copy code
on:
  pull_request:
    types: [closed]
The pull_request event with the closed type is what typically triggers actions when a PR is merged. If this isn't specified correctly, the workflow won't trigger.
Verify the Branch Filters:

Make sure that the workflow is set to run on the main branch or whichever branch you are merging into:
yaml
Copy code
on:
  pull_request:
    branches:
      - main
If the branch filters are incorrect or missing, the workflow may not trigger as expected.

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/136935#discussioncomment-10467941) for reference to why this discussion was closed and locked. Thanks!
